### ISSUE TITLE
Fix issue templates

### ISSUE THREAD

**Issue by whoabuddy** [stacksgov/pm#116]
After #100 the new issue templates are not showing. Two likely reasons:

- content below front matter is commented out, so template is "empty"
- content for name contains a colon on each entry

GitHub help doc: https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository

Could consider a [stale issue template](https://github.com/blockstack/stacks-blockchain/blob/master/.github/stale.yml) workflow like this one as well, but would need community input / possibly a separate issue to track.

**Comment by lrettig** [comment]
Per https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-issue-and-pull-request-templates#issue-templates:

> To be included in the community profile checklist, issue templates must be located in the .github/ISSUE_TEMPLATE folder and contain valid name: and about: YAML front matter.

I think it's the `name` syntax, try putting these in quotes, e.g.,

> name: Agenda: Add Action Item

would become

> name: "Agenda: Add Action Item"

**Comment by friedger** [comment]
Quotes are the solution:
From https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/manually-creating-a-single-issue-template-for-your-repository

![Screenshot from 2020-11-19 10-27-50](https://user-images.githubusercontent.com/1449049/99647386-fd943c00-2a51-11eb-96fd-f1c9a70ee90f.png)

**Comment by whoabuddy** [comment]
Thanks @friedger and @lrettig I knew it would be something simple!
